src/Spectre.Cli/Internal/Modelling/ParameterKind.cs

```diff
@@ -5,10 +5,12 @@ namespace Spectre.Cli.Internal.Modelling
     internal enum ParameterKind
     {
         [Description("flag")]
-        Flag = 0,
+        Flag = 0,
+        [Description("flagvalue")]
+        FlagWithValue = 1,
```

Comment:
Really not liking the "flag" terminology for optional values - as mentioned above it strongly suggests a binary value